### ISSUE TITLE
LGU 2.8.8 doesn't show modded items.

### ISSUE THREAD

**Issue by Dixoul** [Malcolm-Q/LC-LateGameUpgrades#158]
This isn't only affected by LGU. scpecific mods like "buyable shotgun shells" don't appear on the Store item list.
For example. I don't see peeper on the shop and when typing "peeper" says there is nothing. I already checked many times and my mods/libraries are up to date.

Seems like the issue is LethalLib. what version should I use then?

**Comment by WhiteSpike** [comment]
Latest version of LGU and LethalLib installed and I can buy the items mentioned in the issue.

Are you absolutely sure the mods are being loaded?
You can check by looking at either the console it opens upon launch or the file "LogOutput.log" generated in BepInEx folder.

Do these plugin folders indeed contain the .dll file?
I noticed that thunderstore sometimes does not download the .dll file, even though the mod manager say it has.

**Comment by Dixoul** [comment]
Mods loads (as this logs says.) such as LGU and buyable Shells.
Plugins constains their respective .dll and .pdb files.

Also I want to add that I added a JJK scraps theme but I haven't seem them on the map. Even in DanceTools console.
(No, DanceTools don't display lgu/buyable shells items store)
but I saw this message. not really sure what it means but it doesn't seems related to the shop in any way.

**[Info   : LethalLib] LethalLib loaded!!
[Error  : Unity Log] TypeLoadException: Could not load type of field 'LethalLib.Modules.Enemies+<>O:<0>__RegisterLevelEnemies' (0) due to: Could not load file or assembly 'MMHOOK_Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies.
Stack trace:
LethalLib.Plugin.Awake () (at <d94630daeac744428260ff5f2dcb9506>:IL_0078)
UnityEngine.GameObject:AddComponent(Type)
BepInEx.Bootstrap.Chainloader:Start()
UnityEngine.Rendering.HighDefinition.HDRenderPipelineAsset:OnEnable()**


I even tried to uninstall some mods. I tried with only these but still nothing
![imagen_2024-01-13_204336638](https://github.com/Malcolm-Q/LC-LateGameUpgrades/assets/156536181/579e2ab3-e8eb-4da9-8a9d-ef314ae86643)

Any help?
